tests/src/JIT/Performance/CodeQuality/HWIntrinsic/X86/PacketTracer/Color.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using System.Runtime.Intrinsics;
+using static System.Runtime.Intrinsics.X86.Avx;
+using System.Runtime.CompilerServices;
+
+internal struct Color
+{
+    public float R { get; private set; }
```

Comment:
You don't need to declare `private set;`. Readonly auto properties are assignable from the constructor by default.